uint32[] encounter_ids	# array of ids of top matching face encounters
string name		# name of target face, if known
uint32 id		# id of the target face
